### ISSUE TITLE
ResolvePackageNotFound issue

### ISSUE THREAD

**Issue by Siddd25** [v-iashin/BMT#49]
I have recently started to get this issue while extracting the i3d and Vggish packages in the provided Colab. Earlier I was not encountering this issue but now I  am not able to resolve it. Please help to solve this issue as soon as  possible.
![image](https://user-images.githubusercontent.com/98460148/228181488-0bef90a5-c9ce-4aa4-8bda-a781661a1526.png)

**Comment by v-iashin** [comment]
Thanks for letting me know. Indeed, Colab notebook stopped working. It sometimes happens since Google Colab changes its environment frequently.

I did the following changes:
1. updated the conda version that colab installs
```
!wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -q --show-progress
!bash ./Miniconda3-latest-Linux-x86_64.sh -b -f -p /usr/local
```
2. fixed the locale information by adding 
```
import locale
locale.getpreferredencoding = lambda: "UTF-8"
```
3. Removed the build information from env files:
`=p3.7_cuda10.0.130...` from `pytorch=1.2.0` rows in `./submodules/video_features/conda_env_i3d.yml`, `./submodules/video_features/conda_env_vggish.yml`, and `./conda_env.yml`
  
However, when I try to extract VGGish features, it fails with `AttributeError: module 'torch._C' has no attribute '_broadcast_coalesced'`.

Here is the updated notebook: 
[colab_demo_BMT.ipynb.zip](https://github.com/v-iashin/BMT/files/11088759/colab_demo_BMT.ipynb.zip)

I don't have time atm to explore it further. Please let me know if you could fix it.

**Comment by v-iashin** [comment]
This and the follow-up colab errors are fixed in #58
